README.md

```diff
@@ -165,35 +170,38 @@ String name = response.getShop().getName();
 Never use the abstract class directly:
 
 ```java
-// Wrong way, never do this
+// The wrong way (never do this)
 
 AbstractResponse response = ...;
 
 AbstractResponse shop = (AbstractResponse) response.get("shop");
 String name = (String) shop.get("name");
 ```
 
-Again, both of the approaches produce the same result but the former case is safe and requires no casting as it already knows about the expect type.
+Again, both of the approaches produce the same result, but the former case is safe and requires no casting since it already knows about the expected type.
 
 ### The `Node` protocol [⤴](#table-of-contents)
 
-GraphQL shema defines a `Node` interface that declares an `id` field on any conforming type. This makes it convenient to query for any object in the schema given only it's `id`. The concept is carried across to the Buy SDK as well but requeries a cast to the correct type. It is your responsibility to ensure that the `Node` type is of the correct type, otherwise casting to an incorrect type will result in a runtime exception. Given this query:
+GraphQL schema defines a `Node` interface that declares an `id` field on any conforming type. This makes it convenient to query for any object in the schema given only its `id`. The concept is carried across to the Buy SDK as well, but requeries a cast to the correct type. You need to make sure that the `Node` type is of the correct type, otherwise casting to an incorrect type will return a runtime exception.
```

Comment:
"The concept is carried across to the Buy SDK as well, but requeries a cast to the correct type." -- should that be "requeries" or "requires"?